hudi-common/src/main/java/org/apache/hudi/metadata/BaseTableMetadata.java

```diff
@@ -63,7 +72,9 @@
   // Directory used for Spillable Map when merging records
   protected final String spillableMapDirectory;
 
-  protected boolean enabled;
+  protected boolean isMetadataTableEnabled;
+  protected boolean isMetaIndexBloomFilterEnabled = false;
```

Comment:
1. This will be hard to keep updating when more and more indexes are added. 
2. I didnt see where these are set to true.
3. Just enabling them is not enough as the bootstrap should have also taken place.

Reply:
This is the base table. This is set in the derived class during the initialization. More changes around this will come as part of HUDI-3142.